docker-entrypoint.sh

```
@@ -1,15 +1,25 @@
 set -e
 
-# Initialize project if no package.json
-if [ ! -f package.json ]; then
-  echo "Initializing new Bun project..."
-  bun init -y
+echo "--- Running Docker Entrypoint ---"
+
+# Install dependencies if needed
+echo "Checking dependencies..."
+# Ensure npm ci runs if the node_modules volume was just created or is empty
+if [ ! -d "node_modules" ] || [ ! -f "node_modules/.package-lock.json" ] || [ "package-lock.json" -nt "node_modules/.package-lock.json" ]; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, -nt is undefined.

(SC3013)